courses/machine_learning/deepdive2/text_classification/labs/reusable_embeddings.ipynb

```diff
@@ -9,6 +9,8 @@
     "**Learning Objectives**\n",
```

Comment:
<p>Maybe create a TODO here that relates to deploying the model learning objective? </p><p>I know we've done deployment in other labs so this could be reinforcing that concept. </p><p> </p><p>gcloud ai-platform versions create </p><p>--model= # TODO \</p><p>--framework= # TODO  \</p><p>--python-version= # TODO \</p><p>--runtime-version=1.14 \</p><p>--origin=# TODO </p><p>--staging-bucket= # TODO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>